frontend/__mocks__/crds.ts

```diff
@@ -144,23 +147,26 @@ export const testForInvalidVersionsCRD: CustomResourceDefinitionKind = {
       singular: 'testresource',
       listKind: 'TestResourceList',
     },
+    scope: 'Namespaced',
   },
 };
 
 export const testForNoVersionsCRD: CustomResourceDefinitionKind = {
-  apiVersion: 'apiextensions.k8s.io/v1beta1',
+  apiVersion: 'apiextensions.k8s.io/v1',
   kind: 'CustomResourceDefinition',
   metadata: {
     name: 'testresources.testapp.coreos.com',
   },
   spec: {
     group: 'testapp.coreos.com',
     version: 'correct-version',
```

Comment:
`version` is not a valid field for API version `apiextensions.k8s.io/v1`. We will also likely need to update the test or tests that use this object.
```suggestion
```